plugin.xml

```diff
@@ -100,15 +100,15 @@
         <source-file src="src/android/CleverTapPlugin.java" target-dir="src/com/clevertap/cordova/" />
 
         <framework src="com.google.firebase:firebase-messaging:$FIREBASE_MESSAGING_VERSION" />
-        <framework src="com.clevertap.android:clevertap-android-sdk:4.2.0" />
-        <framework src="com.github.bumptech.glide:glide:4.11.0" />
-        <framework src="androidx.appcompat:appcompat:1.2.0" />
+        <framework src="com.clevertap.android:clevertap-android-sdk:4.4.0"/>
+        <framework src="com.github.bumptech.glide:glide:4.12.0"/>
+        <framework src="androidx.appcompat:appcompat:1.3.1"/>
         <framework src="androidx.core:core:1.3.0" />
-        <framework src="androidx.fragment:fragment:1.1.0" />
-        <framework src="androidx.recyclerview:recyclerview:1.1.0" />
+        <framework src="androidx.fragment:fragment:1.3.6"/>
+        <framework src="androidx.recyclerview:recyclerview:1.2.1"/>
         <framework src="androidx.viewpager:viewpager:1.0.0" />
-        <framework src="com.google.android.material:material:1.2.1" />
-        <framework src="com.android.installreferrer:installreferrer:2.1" />
+        <framework src="com.google.android.material:material:1.4.0"/>
+        <framework src="com.android.installreferrer:installreferrer:2.2"/>
 
         <hook type="after_plugin_add" src="scripts/androidAfterPluginAdd.js" />
 
```

Comment:
check for support of fcm push ?

check for android 12 working without push templates?

check for app working when push templates is added directly in android/ build.gradle or properties file

Reply:
where to call `CleverTapAPI.setNotificationHandler((NotificationHandler)new PushTemplateNotificationHandler());`?